qemu/tests/qemu_guest_agent.py

```diff
@@ -326,6 +326,18 @@ def gagent_check_sync(self, test, params, env):
         error_context.context("Check guest agent command 'guest-sync'", logging.info)
         self.gagent.sync()
 
+    @error_context.context_aware
+    def gagent_check_ping(self, test, params, env):
+        """
+        Execute "guest-ping" command to guest agent
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environmen.
+        """
+        error_context.context("Check guest agent command 'guest-ping'", logging.info)
+        self.gagent.verify_responsive()
+
```

Comment:
gagent_check_ping() is almost the same as existed method gagent_verify(), and basically all cases will run guest-ping case, so I think there is no need to add the special guest-ping case.

Reply:
We need a basic and simple qemu-ga case to check if qemu-ga works.

Reply:
I mean if you need a basic and simple qemu-ga case, you can directly use existed method gagent_verify(). This method is to execute guest-ping.